tests/test_acasclient.py

```diff
@@ -2367,6 +2367,203 @@ def test_50_delete_ls_thing(self):
+                    "sdfProperty": "Lot Solution Amount Units"
+                },
+                {
+                    "dbProperty": "Lot Supplier",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Lot Source"
+                },
+                {
+                    "dbProperty": "Lot Supplier ID",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Source ID"
+                },
+                {
+                    "dbProperty": "CAS Number",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "CAS"
+                },
+                {
+                    "dbProperty": "Project",
+                    "defaultVal": self.global_project_code,
+                    "required": True,
+                    "sdfProperty": "Project Code Name"
+                },
+                {
+                    "dbProperty": "Parent Common Name",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Name"
+                },
+                {
+                    "dbProperty": "Parent Stereo Category",
+                    "defaultVal": "Unknown",
+                    "required": True,
+                    "sdfProperty": None
+                },
+                {
+                    "dbProperty": "Parent Stereo Comment",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Structure Comment"
+                },
+                {
+                    "dbProperty": "Lot Is Virtual",
+                    "defaultVal": "False",
+                    "required": False,
+                    "sdfProperty": "Lot Is Virtual"
+                },
+                {
+                    "dbProperty": "Lot Supplier Lot",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Sample ID2"
+                },
+                {
+                    "dbProperty": "Parent Alias",
+                    "defaultVal": None,
+                    "required": False,
+                    "sdfProperty": "Parent Alias"
+                }
+            ]
+        }
+        response = self.client.register_sdf_request(request)
+        self.assertIn('reportFiles', response[0])
+        self.assertIn('summary', response[0])
+        self.assertIn('Number of entries processed', response[0]['summary'])
+
+        # Setup constants
+        corp_name = "CMPD-0000051"
+        alias_one = "First Alias"
+        alias_two = "Second Alias"
+        alias_three = "Third Alias"
+        alias_four = "Fourth Alias"
+        alias_five = "Fifth Alias"
+
+        # Get aliases
+        aliases = self.client.get_parent_aliases(corp_name)
+        self.assertEqual(len(aliases), 3)
```

Comment:
@dalejerikson  Could you add a test case where we load the same file a second time, and confirm we still only get 3 aliases? Just to test that the string comparison logic (which i see you fixed on the acas-roo-server PR) is working as expected.

Reply:
Will do!

Reply:
Completed